docs/native/src/faq/filetypes.rst

```diff
@@ -0,0 +1,16 @@
+What file types are supported in |Office iOS|?
```

Comment:
A link to the conversion is good as an addition. 
however, unless we have pages covering all three apps, we should have the list here. We get asked this question pretty frequently (about which file format we support) and if we don't provide guidance here, partners might only do the promotion for the main file types which would be bad for us.